x/ccv/child/keeper/genesis.go

```diff
@@ -57,7 +89,12 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) *types.GenesisState {
 	}
 
 	if channelID, ok := k.GetParentChannel(ctx); ok {
-		gs := types.NewRestartGenesisState(channelID, nil)
+		clientID, ok := k.GetParentClient(ctx)
+		if !ok {
+			panic("parent client does not exist")
+		}
+		// ValUpdates must be filled in off-line
```

Comment:
I'm not following this comment

Reply:
Oh, they need to paste the latest provider valset updates in themselves, right? Maybe some time we can go over the exact procedure involved in doing a consumer chain genesis export and restart